test/BinTestCases.test.js

```diff
@@ -53,6 +55,7 @@ function findTestsRecursive(readPath) {
 	const isAnyTests = entries.indexOf("stdin.js") !== -1;
 
 	const folders = entries
+		.filter(entry => !migratedTests.includes(entry))
```

Comment:
I haven't tested if this even works... Might be best to migrate all tests at once, should be fairly straighforward once you get the hang of it